src/components/Header.tsx

```diff
@@ -0,0 +1,18 @@
+<header className="todoapp__header">
```

Comment:
?
I understand , but keep in mind , GH that is place where your code should be clean , you can keep this files locally but not on GH